source/ptreferential/tests/ptref_ng_test.cpp

```diff
@@ -75,6 +75,9 @@ BOOST_AUTO_TEST_CASE(parse_pred) {
     assert_expr(
         R"#(vehicle_journey . has_code_type ( external_code ) )#",
         R"#(vehicle_journey.has_code_type("external_code"))#");
+    assert_expr(
+        R"#(route . has_code_type ( forward ) )#",
```

Comment:
"wrong" copy/paste (same below).
It tests the same method as above on another object, but it's probably not what you wanted.